packages/compiler-cli/test/ngtsc/template_mapping_spec.ts

```diff
@@ -118,7 +118,7 @@ describe('template source-mapping', () => {
         });
         expect(mappings).toContain({
           source: '[attr]="greeting + name"',
-          generated: 'i0.ɵɵelementProperty(0, "attr", i0.ɵɵbind((ctx.greeting + ctx.name)))',
+          generated: 'i0.ɵɵproperty("attr", (ctx.greeting + ctx.name))',
```

Comment:
Same here - are the extra parens necessary?

```suggestion
          generated: 'i0.ɵɵproperty("attr", ctx.greeting + ctx.name)',
```

Reply:
Oops. probably missed during a find a replace :)

Reply:
Update: Will be fixing this in a follow-up.